scripts/common/proxy.sh

```diff
@@ -1,14 +1,12 @@
 
 function proxy_bootstrap() {
-    if [ "$AIRGAP" = "1" ]; then
-        return
-    fi
-
     if [ -n "$HTTP_PROXY" ]; then
         ENV_PROXY_ADDRESS="$HTTP_PROXY"
+        export https_proxy="$HTTP_PROXY"
         printf "The installer will use the proxy at '%s' (imported from env var 'HTTP_PROXY')\n" "$ENV_PROXY_ADDRESS"
     elif [ -n "$http_proxy" ]; then
         ENV_PROXY_ADDRESS="$http_proxy"
+        export https_proxy="$HTTP_PROXY"
```

Comment:
should this be `$http_proxy` instead?

Reply:
It has to be https_proxy or curl ignores it for https urls

Reply:
i meant `$http_proxy` instead of `$HTTP_PROXY`